buildkite/scripts/tests/rosetta-integration-tests.sh

```diff
@@ -172,7 +191,11 @@ mina daemon \
   --rest-port ${MINA_GRAPHQL_PORT} \
   --run-snark-worker "$SNARK_PRODUCER_PK" \
   --seed \
-  --demo-mode
+  --demo-mode \
+  > daemon-stdout.log 2> daemon-stderr.log < /dev/null &
```

Comment:
why read from /dev/null as stdin?

Reply:
This is not necessary here, or probably anywhere. It was suggested by the AI when I was paranoid about removing the `--background` flag because maybe the daemon would try to read something from stdin (we actually do this with e.g. key files.